### ISSUE TITLE
Duplicated test execution while using `cucumber.features` property

### ISSUE THREAD

**Issue by fslevoaca-ionos** [cucumber/cucumber-jvm#2761]
It seems tests are executed twice via the `cucumber.features` property.

Project to reproduce the issue:  https://github.com/fslev/cucumber-showcase

1. Run Cucumber test scenarios using the `cucumber.features` property
```
mvn clean test -Dcucumber.features=classpath:io/example/Feature1.feature:3
```
Actual:
```
Running scenario: Feature 1 - Scenario 1_1
Running scenario: Feature 1 - Scenario 1_1
```
 
`Scenario 1_1` is executed twice.

While using tags:
```
mvn clean verify -Dcucumber.filter.tags="@1"
```
... scenario is executed correctly, only once.

Thanks

**Comment by mpkorstanje** [comment]
In your project Cucumber gets two test discovery/execution requests. One when Maven makes a discovery request. And a second one through the JUnit Platform Suite engine. Normally the request made by Maven fails to discover any tests because Maven only requests the discovery of classes. However by setting the  `cucumber.features` it will discover something.

So when using `cucumber.features` you must ensure that there is only a single execution of the Cucumber Engine.

Ultimately this comes down to a lack of proper support for the JUnit Platform in Maven (https://github.com/junit-team/junit5/issues/2849) and IDEA. If that existed you would not have to use the JUnit Platform Suite Engine to run Cucumber.

Though I'm surprised you didn't see this warning:

https://github.com/cucumber/cucumber-jvm/blob/b53204e60447551a916d4a7b7e2856e8167fbf76/cucumber-junit-platform-engine/src/main/java/io/cucumber/junit/platform/engine/DiscoverySelectorResolver.java#L39-L44


Duplicate of #2567.
